extensions/kafka-streams/deployment/src/main/java/io/quarkus/kafka/streams/deployment/KafkaStreamsBuildTimeConfig.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.kafka.streams.deployment;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "kafka-streams", phase = ConfigPhase.BUILD_TIME)
+public class KafkaStreamsBuildTimeConfig {
+
+    /**
+     * Whether or not an health check is published in case the smallrye-health extension is present (default to true).
+     */
+    @ConfigItem(name = "health.enabled", defaultValue = "true")
```

Comment:
I think this should be a String allowing the following values:

- true
- false
- the name of the check to enable as there is two different check

=> If you don't want to implent it now, please open a followup issue for this

Reply:
issue #5572 created